### PR TITLE
354/update and fix

### LINKED ISSUE #354: Update dependencies to resolve build error with eslint

### PR DESCRIPTION
**Pull request by pyphilia**
close #354

### CODE REVIEW

File: scripts/setup.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ const env = JSON.stringify({
   GOOGLE_API_KEY,
   GOOGLE_ANALYTICS_ID,
   LOGGING_LEVEL,
+  HELLO: 'er',
```
Comment (juancarlosfarah) [inline scripts/setup.js:19]: Do we really need these?

File: src/components/common/WelcomeContent.js [hunk 2]
```diff
@@ -102,7 +102,6 @@ class WelcomeContent extends Component {
           {t('Welcome to Graasp Desktop')}
         </Typography>
         <ButtonGroup
-          disableElevation
```
Comment (juancarlosfarah) [inline src/components/common/WelcomeContent.js:105]: Is this no longer needed?

File: src/components/dashboard/ActionBarChart.js [hunk 3]
```diff
@@ -35,12 +35,9 @@ class ActionBarChart extends PureComponent {
     theme: PropTypes.shape({
       palette: PropTypes.shape({
         type: PropTypes.string.isRequired,
-        primary: PropTypes.arrayOf(PropTypes.string).isRequired,
+        primary: PropTypes.shape().isRequired,
```
Comment (juancarlosfarah) [inline src/components/dashboard/ActionBarChart.js:38]: Missing `{}`?

File: src/components/dashboard/ActionLineChart.js [hunk 4]
```diff
@@ -35,12 +35,9 @@ class ActionLineChart extends PureComponent {
     theme: PropTypes.shape({
       palette: PropTypes.shape({
         type: PropTypes.string.isRequired,
-        primary: PropTypes.arrayOf(PropTypes.string).isRequired,
+        primary: PropTypes.shape().isRequired,
```
Comment (juancarlosfarah) [inline src/components/dashboard/ActionLineChart.js:38]: Missing `{}`?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good! Just some minor changes.

**Comment by pyphilia** [comment]
Thank you for your review, I took this opportunity to close #348 as well.

**Review by juancarlosfarah — APPROVED** [review]

**Pull request merged**